services/common/src/interfaces/amendment.interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface IAmendment {
+  permit_amendment_guid: string;
+  permit_amendment_status_code: string;
+  permit_amendment_type_code: string;
+  permit_ammendment_id: string;
```

Comment:
Extra `m`? 😅

Reply:
Ugh, didn't see your comments until after I had merged.  I'll do a quick PR to fix these things and keep an eye on things.  I'm pretty sure the likelyhood of it blowing up is small, but perhaps I'll just monitor for now.

Reply:
Totally up to you! Monitoring sounds good. It's always easy enough to do a revert if needed